code/game/machinery/vending/loadout.dm

```diff
@@ -701,10 +701,10 @@
 		/obj/item/storage/box/fluff/swimsuit/highclass = 3,
 		/obj/item/storage/box/fluff/swimsuit/latex = 3,
 		/obj/item/storage/box/rainponcho = 5,
-		/obj/item/clothing/under/redpyjamas = 5,
-		/obj/item/clothing/under/redpyjamas_fem = 5,
-		/obj/item/clothing/under/bluepyjamas = 5,
-		/obj/item/clothing/under/bluepyjamas_fem = 5,
+		/obj/item/clothing/under/pj/red = 5,
```

Comment:
nitpick

/pajamas 
don't overly-aggressively contract paths, they're meant to be readable